apps/fluent-tester/src/FluentTester/FluentTester.tsx

```diff
@@ -204,16 +204,35 @@ export const FluentTester: React.FunctionComponent<FluentTesterProps> = (props:
     );
   };
 
-  return (
-    <RootView style={themedStyles.root}>
-      {enableSinglePaneView ? <MobileHeader /> : <Header />}
+  const TesterContent: React.FunctionComponent<{}> = () => {
+    return (
+      <RootView style={themedStyles.root}>
+        {enableSinglePaneView ? <MobileHeader /> : <Header />}
 
-      <HeaderSeparator />
+        <HeaderSeparator />
 
-      <View style={fluentTesterStyles.testRoot}>
-        {enableSinglePaneView ? <MobileTestList /> : <TestList />}
-        {enableSinglePaneView ? <MobileTestComponentView /> : <TestComponentView />}
-      </View>
-    </RootView>
-  );
+        <View style={fluentTesterStyles.testRoot}>
+          {enableSinglePaneView ? <MobileTestList /> : <TestList />}
+          {enableSinglePaneView ? <MobileTestComponentView /> : <TestComponentView />}
+        </View>
+      </RootView>
+    );
+  };
+
+  const Win32TesterContent: React.FunctionComponent<{}> = () => {
+    return (
+      <FocusTrapZone style={themedStyles.root}>
+        {enableSinglePaneView ? <MobileHeader /> : <Header />}
```

Comment:
**Code Duplication**

We can avoid the code duplication here for everything within the <FocusTrapZone /> and <RootView /> above. Maybe make a functional component just for the content within those two sections?

So something like:
  const TestContent: React.FunctionComponent<{}> = () => {
    return (
      <View>
        {enableSinglePaneView ? <MobileHeader /> : <Header />}

        <HeaderSeparator />

        <View style={fluentTesterStyles.testRoot}>
          {enableSinglePaneView ? <MobileTestList /> : <TestList />}
          {enableSinglePaneView ? <MobileTestComponentView /> : <TestComponentView />}
        </View>
      </View>
    );
  };

  const TesterWrapper: React.FunctionComponent<{}> = () => {
    return (
      <View>
        {Platform.OS === ('win32' as any) ? (
          <FocusTrapZone style={themedStyles.root}>
            <TestContent />
          </FocusTrapZone>
        ) : (
          <RootView style={themedStyles.root}>
            <TestContent />
          </RootView>
        )}
      </View>
    );
  };

Just a thought to clean up the code a bit

Reply:
RootView itself was just a custom wrapper to let there be a `SafeAreaView` at the top level instead of a `View` for iOS. I think it may be completely fine for us to add an extra case to the platform select of `RootView` to be FocusTrapZone on win32, unless FocusTrapZone itself must be wrapped by a View

Reply:
Yeah, definitely. I wanted to clean this up but got busy and wanted to push out a working change. @Saadnajmi I tried adding another case to platform select but I couldn't get the syntax right for the win32 case... do you know how I might do that?

Reply:
Does this not work?
`
  const RootView = Platform.select({
    ios: SafeAreaView,
    win32: FocusTrapZone,
    default: View,
  });
  `

Reply:
No, because win32 is not part of Platform (only android, ios, windows, macos, and default). I think usually we do "win32 as any" but I'm not sure how to make it work in this case.

Reply:
:') That's frustrating. I guess then Sammy's suggestion is the next best thing.

Reply:
I did a modified version of Sammy's suggestion (just had to add some styling to the two additional View's). Thanks for the feedback!

Reply:
Awesome! I'll probably revisit the whole RootView thing later since it's not as extensible as I hoped.